src/connector.js

```diff
@@ -742,6 +784,48 @@ function initEngine() {
 	}
 }
 
+function searchBoxArrowKeyUp() {
+	if ( suggestionsElement.hidden ){
+		return;
+	}
+
+	if ( !activeSuggestion || activeSuggestion <= 1 )
+		activeSuggestion = searchBoxState.suggestions.length;
+	else
+		activeSuggestion -= 1;
+
+	updateSuggestionSelection();
+}
+
+function searchBoxArrowKeyDown() {
+	if ( suggestionsElement.hidden ){
+		return;
+	}
+
+	if ( !activeSuggestion || activeSuggestion >= searchBoxState.suggestions.length )
+		activeSuggestion = 1;
+	else
+		activeSuggestion += 1;
+
+	updateSuggestionSelection();
+}
```

Comment:
Suggestion: For optimization purposes, I believe these functions could be combined into one, with a parameter for the `key`.